doc/nrf/images/zigbee_signal_handler_07_idle.uml

```diff
@@ -0,0 +1,18 @@
+@startuml
+
+:zboss_main_loop_iteration();
+
+if (scheduler_queue_empty?) then (yes)
+if ((inactivity_time > sleep_threshold) and (rx_off_when_idle)) then (yes)
```

Comment:
To be improved:
 - `zb_osif_go_idle` left unimplemented in ZBOSS
 - `ZB_COMMON_SIGNAL_CAN_SLEEP` is even if `rx_off_when_idle` if set to false.